src/plugins/console/public/application/containers/editor/legacy/console_editor/editor_output.tsx

```diff
@@ -85,6 +85,11 @@ function EditorOutputUI() {
             if (readOnlySettings.tripleQuotes && isJSONContentType(contentType)) {
               return safeExpandLiteralStrings(value as string);
             }
+            if (contentType.includes('application/vnd.mapbox-vector-tile')) {
```

Comment:
nit: we could extract this into a function that's something like `isMapboxVectorTile` to keep the pattern as we already have with `isJSONContentType`